redaxo/src/core/pages/system.settings.php

```diff
@@ -88,6 +88,42 @@
             $success = rex_i18n::msg('info_updated');
         }
     }
+} elseif ('updateeditor' === $func) {
+    $editor = rex_post('editor', [
+        ['name', 'string', null],
+        ['basepath', 'string', null],
+        ['update_cookie', 'bool', false],
+        ['delete_cookie', 'bool', false],
+    ]);
+
+    $cookieOptions = ['samesite' => 'strict'];
+
+    if ($editor['delete_cookie']) {
+        rex_response::clearCookie('editor', $cookieOptions);
```

Comment:
Die save/clear logik ggf in die editor klasse verlagern, damit der core nicht annahmen treffen muss wo gespeichert wird?

Reply:
Wie würdest du die Methoden gestalten?
Eher irgendwie sowas wie `update($editor, $basePath)` und `overwriteViaCookie($editor, $basePath)`? Oder eher jeweils separate Methoden für Editor und BasePath?

Wobei ich es eigentlich auch so ok finde, wie es ist. Also wegen mir brauchen wir hier aktuell keine public Api und mich stört es nicht, wenn der Core hier in der Page es selbst an die korrekten Stellen legt.